content/applications/finance/accounting/taxes/avatax/avatax_use.rst

```diff
@@ -100,3 +100,55 @@ value. To recalculate the taxes, click :guilabel:`Compute taxes using Avatax`.
 
 .. tip::
    Tax calculation can even be done on negative subtotals and credit notes.
+
+Logging
+=======
+
+It is possible to log Avalara/*AvaTax* actions in Odoo for further analysis or verification of
+functionality. Logging is accessible through the *AvaTax* settings.
+
+To start logging *AvaTax* actions, first, navigate to the :menuselection:`Accounting app -->
+Configuration --> Settings`. Then, in the :guilabel:`Taxes` section, under the :guilabel:`AvaTax`
+settings, click :guilabel:`Start logging for 30 minutes`. Upon starting the log process, Odoo will
+log all Avalara/*AvaTax* actions performed in the database.
```

Comment:
```suggestion
To start logging *AvaTax* actions, first, navigate to the :menuselection:`Accounting app -->
Configuration --> Settings`.

Then, in the :guilabel:`Taxes` section, under the :guilabel:`AvaTax` settings, click
:guilabel:`Start logging for 30 minutes`.

Upon starting the log process, Odoo will log all Avalara/*AvaTax* actions performed in the database.
```
just to break up an otherwise "chunky" paragraph